File: examples/src/Slides/Background/GetBackgroundEffectiveValues.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.slides.Background
{
    class GetBackgroundEffectiveValues
    {

        public static void Run()
        {
            #ExStart:GetBackgroundEffectiveValues
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Background()

            # Instantiate the Presentation class that represents the presentation file
            Presentation pres = new Presentation(dataDir + "SamplePresentation.pptx")

            IBackgroundEffectiveData effBackground = pres.slides[0].Background.GetEffective()

            if (effBackground.fill_format.fill_type == slides.FillType.SOLID)
                print("Fill color: " + effBackground.FillFormat.SolidFillColor)
            else
                print("Fill type: " + effBackground.FillFormat.FillType)

            #ExEnd:GetBackgroundEffectiveValues
        }

    }
}

File: examples/src/slides/background/get_background_effective_values.py
import aspose.slides as slides


def get_background_effective_values(global_opts):
    # Instantiate the Presentation class that represents the presentation file
    with slides.Presentation(global_opts.data_dir + "background.pptx") as pres:
        effective_background = pres.slides[0].background.get_effective()
        if effective_background.fill_format.fill_type == slides.FillType.SOLID:
            print("Fill color: " + str(effective_background.fill_format.solid_fill_color))
        else:
            print("Fill type: " + str(effective_background.fill_format.fill_type))
